rhizome/host/lib/cert_server_setup.rb

```diff
@@ -130,11 +131,13 @@ def stop_and_remove_service
     FileUtils.rm_f(service_file_path)
   end
 
+  def create_cert_folder
+    FileUtils.mkdir(cert_folder)
+  rescue Errno::EEXIST
+  end
+
   def put_certificate(cert_payload, cert_key_payload)
-    begin
-      FileUtils.mkdir(cert_folder)
-    rescue Errno::EEXIST
-    end
+    create_cert_folder
     safe_write_to_file(cert_path, cert_payload)
```

Comment:
Since cert server is up already, it's now possible that a user request comes at at time where cert is written already but key is not written, yet. Request to get the cert succeeds but request to get the key fails. I assume that shouldn't be a big deal, though.